File: pinyougou-manager-web/src/main/webapp/js/service/specificationService.js
app.service('specificationService',function ($http) {
    this.search=function (pageNum, pageSize, searchEntity) {
        return $http.post("../specification/search?pageNum=" + pageNum + "&pageSize=" + pageSize, searchEntity)
    }

    this.add = function (entity) {
        return $http.post('../specification/add', entity);
    }

    this.update = function (entity) {
        return $http.post('../specification/update', entity);
    }
    this.findOne = function (id) {
        return $http.post('../specification/findOne?id=' + id);
    }
    this.dele = function (selectId) {
        return $http.get('../specification/dele?ids=' + selectId);
    }
})

File: pinyougou-manager/pinyougou-manager-interface/src/main/java/com/pinyougou/manager/service/SpecificationService.java
package com.pinyougou.manager.service;

import com.pinyougou.common.PageResult;
import com.pinyougou.pojo.SpecificationVo;
import com.pinyougou.pojo.TbSpecification;

import java.util.List;
import java.util.Map;

public interface SpecificationService {
    PageResult search(Integer pageNum, Integer pageSize, TbSpecification tbSpecification);

    void add(SpecificationVo specificationVo);

    SpecificationVo findOne(Long id);

    void update(SpecificationVo specificationVo);

    void dele(Long[] ids);

    List<Map> findSpecList();
}
